imagetest/test_suites/ssh/setup.go

```diff
@@ -0,0 +1,42 @@
+package ssh
+
+import (
+	"io/ioutil"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest"
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+// Name is the name of the test package. It must match the directory name.
+var Name = "ssh"
+
+const user = "test-user"
+
+// TestSetup sets up the test workflow.
+func TestSetup(t *imagetest.TestWorkflow) error {
+
+	vm, err := t.CreateTestVM("vm")
+	if err != nil {
+		return err
+	}
+
+	vm.AddMetadata("block-project-ssh-keys", "true")
+
+	if err := utils.GenerateSSHKeyPair(user); err != nil {
```

Comment:
test author must call this, then read the pubkey file, then set up metadata... it would be better if we have one helper like vm.AddTestUser(username) that returns the data needed for the test author.

Reply:
Move the function to fixture and make it as a method for vm.